op-service/sources/l1_beacon_client.go

```diff
@@ -298,9 +298,13 @@ func blobsFromSidecars(blobSidecars []*eth.BlobSidecar, hashes []eth.IndexedBlob
 		}
 
 		if !skipBlobVerification {
-			// confirm blob data is valid by verifying its proof against the commitment
-			if err := eth.VerifyBlobProof(&sidecar.Blob, kzg4844.Commitment(sidecar.KZGCommitment), kzg4844.Proof(sidecar.KZGProof)); err != nil {
-				return nil, fmt.Errorf("blob at index %d failed verification: %w", i, err)
+			// recompute the commitment from the blob data and check it matches the commitment from the sidecar
+			recomputedCommitment, err := kzg4844.BlobToCommitment(sidecar.Blob.KZGBlob())
```

Comment:
We probably want to emit metrics for this so. Recomputing the commitments is supposed to be much slower than verifying proofs the way we were doing it before. I'm assuming it's still fast enough that it won't have any affect on the nodes, but collecting timing data will help us be certain.

Reply:
I ran some benchmarks on my laptop, and computing a commitment takes around 5ms where verifying a proof takes 1.5ms.

Reply:
It takes 6ms to (re)-compute a blob proof, so that isn't looking competitive actually.

Reply:
I agree, ~5ms (and net +~3.5ms) per blob is negligible compared to all the other things that are happening in either an op-program run or a verified node during derivation. I don't think we need to add metrics given the existing benchmarks.